Async-Context-Definitions.md

```diff
@@ -403,60 +403,114 @@ and the top of the desired long call-stack.
 
 **TODO** add an example with sample code etc.
 
-## Asynchronous Operation Metadata
-In the previous sections we focused solely on tracking and emitting 
-information on the structure of the asynchronous call graph. However, most 
-applications, including our samples above, are interested in more than just the raw structure of this graph. While we cannot, and would not want to, identify 
-all possible data that could be needed and write it out to our log we can 
-(1) select core of commonly useful information to include and (2) add a timestamp 
-that is shared with user logging code to allow the correlation of custom user 
-log data with the asynchronous event data we write.
+## Asynchronous Object Lifecycle

```

Comment:
My .02 here is this is useful, but it risks distracting from the core message defining async context.